docs/swarm/admin_guide.md

```diff
@@ -0,0 +1,243 @@
+<!--[metadata]>
++++
+aliases = [
+"/engine/swarm/manager-administration-guide/"
+]
+title = "Swarm Manager Administration Guide"
+description = "Manager administration guide"
+keywords = ["docker, container, cluster, swarm, manager, raft"]
+advisory = "rc"
+[menu.main]
+identifier="manager_admin_guide"
+parent="engine_swarm"
+weight="12"
++++
+<![end-metadata]-->
+
+# Administer and maintain a swarm of Docker Engines
+
+When you run a swarm of Docker Engines, **manager nodes** are the key components
+for managing the cluster and storing the cluster state. It is important to understand
+some key features of manager nodes in order to properly deploy and maintain the
+swarm.
+
+This article covers the following swarm administration tasks:
+
+* [Add Manager nodes for fault tolerance](#add-manager-nodes-for-fault-tolerance)
+* [Distributing manager nodes across regions](#distribute-manager-nodes-across-datacenter-regions)
+* [Running manager-only nodes](#run-manager-only-nodes)
+* [Backing up the cluster state](#back-up-the-cluster-state)
+* [Monitoring the swarm health](#monitor-swarm-health)
+* [Recovering from disaster](#recover-from-disaster)
+
+Refer to [How swarm mode nodes work](how-swarm-mode-works/nodes.md)
+for a brief overview of Docker Swarm mode and the difference between manager and
+worker nodes.
+
+## Operating manager nodes in a swarm
+
+Swarm manager nodes use the [Raft Consensus Algorithm](raft.md) to manage the
+cluster state. You only need to understand some general concepts of the raft
+implementation in order to manage a swarm.
+
+There is no limit to scale for manager nodes. The decision about how many
+manager nodes to implement is a trade-off between performance and
+fault-tolerance. Adding manager nodes to a swarm makes the swarm more
+fault-tolerant. However, additional manager nodes reduce write performance
+because more nodes must acknowledge proposals to update the cluster state.
+This means more network round-trip traffic.
+
+Raft requires a majority of members, also called a quorum of `(N/2)+1`, to
+agree on proposed updates to the cluster. A quorum must also agree on node
+additions and removals. Membership operations are subject to the same
+constraints as state replication.
+
+## Add manager nodes for fault tolerance
+
+You should maintain an odd number of managers in the swarm to support manager
+node failures. Having an odd number of nodes ensures that during a network
+partition, a quorum remains available to process requests.
```

Comment:
Good point, thought about including that but kept the explanation simple for the first draft. I'll make the change.